docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -388,3 +422,970 @@ GET /api/proposals
+      {
+        "service_id": "gameroom_bubba-node-000",
+        "service_type": "scabbard",
+        "allowed_nodes": [ "bubba-node-000" ],
+        "arguments": {
+          "peer_services": [ "gameroom_acme-node-000" ],
+          "admin_keys": [
+            <acme gameroomd public key>
+            ]
+          }
+        }
+      ],
+      "circuit_management_type": "gameroom",
+      "application_metadata":​ <metadata bytes defined by the application>​,
+      "persistence": "Any",
+      "durability": "None",
+      "routes": "Any",
+    },
+    "vote_record": [{}],
+    "requester": "public_key_of_requester",
+    "requester_node_id": "acme-node-000"
+  }
+}
+```
+
+#### I-2.8. Gameroom daemons write notification to Gameroom database
+
+When each Gameroom application authorization handler receives the gameroom
+proposal on the WebSocket connection, it parses the information and adds it to
+several tables in the Gameroom daemon's local database: `​gameroom​`, ​
+`gameroom_member​`, `​gameroom_service​`, `gameroom_proposal​`, and
+`​gameroom_notification​`.
+
+##### I-2.8.1. New `gameroom` table entry
+
+First, the Gameroom application authorization handler adds a new entry to the
+​gameroom​ table. This table contains the information about the circuit
+definition, including the data that was passed in the ​application_metadata​ field.
+
+``` sql
+CREATE TABLE IF NOT EXISTS  gameroom (
+  circuit_id TEXT PRIMARY KEY,
+  authorization_type TEXT NOT NULL,
+  persistence TEXT NOT NULL,
+  routes TEXT NOT NULL,
+  durability TEXT NOT NULL,
+  circuit_management_type TEXT NOT NULL,
+  alias TEXT NOT NULL,
+  status TEXT NOT NULL,
+  created_time TIMESTAMP NOT NULL,
+  updated_time TIMESTAMP NOT NULL,
+);
+```
+
+* `circuit_id`, `authorization_type`, `persistence`, `routes`, `durability`,​ and
+  `circuit_management_type` ​are extracted directly from the circuit proposal
+  message that is received from the Splinter daemon.
+
+* `alias​​` is extracted by deserializing the application metadata in the circuit
+  proposal message. The alias is the gameroom name that Alice entered when
+  creating the gameroom in the Acme UI.
+
+* `status​` identifies the current status of the gameroom. In this case, it is
+  set to ​pending because the proposal to create this gameroom has not yet been
+  accepted.
+
+* `created_time` is when the gameroom entry was introduced in the table.
+
+* `updated_time` is when the gameroom entry was last updated.
+
+At the end of the operation, the `​gameroom​` table looks like this:
+
+| `circuit_id` | `authorization_type` |
+| :--- | :--- |
+|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|`Trust`|
+
+| `persistence` | `routes` | `durability` | `circuit_management_type` |
+| :--- | :--- | :--- | :--- |
+| `Any` | `Any` | `None` | `gameroom` |
+
+
+| `alias` | `status` | `created_time` | `updated_time` |
+| :--- | :--- | :--- | :--- |
+|`Acme + Bubba`|`pending`|`<time entry was created>`|`<time entry was updated>`|
```

Comment:
These tables aren't rendering correctly. There's no format difference between the "header" rows and the content rows, and the space between tables isn't being shown. I wonder if HTML tags (\<table>, \<tr>, \<td>, etc.) would work better?

But tables are a pain - and might never look good on a small screen. 

Maybe this information should be presented in a different way, such as a code block with spaces to align information?

Reply:
Updated all of the tables to be generated with html